com.beeper.Beeper.yaml

```diff
@@ -0,0 +1,81 @@
+#This package is not affiliated to Beeper Team
+app-id: com.beeper.Beeper
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: startbeeper
+separate-locales: false
+finish-args:
+  # Version required to use the document-portal v4
+  - --require-version=1.7.0
+  # Required due to being a GUI application
+  - --socket=x11
+  # Required to make sure x11 performance is achived on all platforms
+  # At least that's what the legends tell. it might be worth experimenting
+  # with dropping this permission.
+  - --share=ipc
+  # Required for experimental wayland support
+  - --socket=wayland
```

Comment:
if wayland support is 'experimental' it should be disabled. Otherwise line 14 is 'fallback-x11'.

Reply:
Alright, I'll do the rest tomorrow because I just got back home and I'm gonna sleep